tools/gather_outputs.sh

```diff
@@ -50,13 +50,15 @@ for file in *dump.*.000.${expnr}.nc ; do
         dumps=${file%.000.${expnr}.nc}
 
         if [ ${dumps:0:9} == "fielddump" ]; then
-            ymparam="v,tau_y,ym"
+            #ymparam="v,tau_y,ym"
```

Comment:
Note if 'tau_x' is specified as a fielddump variable, this line needs to be used, but now that we have stress output on facets this is probably needed less, so let's keep it this way.

Reply:
Modified it appropriately. Kept in this way

ymparam="v,tau_y,ym"
#ymparam="v,ym"